filament/src/materials/ssao/ssaoUtils.fs

```diff
@@ -13,5 +13,16 @@ highp float sampleDepthLinear(const sampler2D depthTexture, const vec2 uv, float
     return linearizeDepth(textureLod(depthTexture, uv, lod).r, depthParams);
 }
 
+highp vec2 ssFromUv(highp vec2 uv) {
```

Comment:
I read this as "screenSpace", it's not really screen space, it's still a UV. Not sure what a better name would be. screenUvFromUv()?

Reply:
Agreed, I also thought it was "screen-space" -- but it's like a normalized screenspace, isn't that "NDC", Normalized device coordinate?
This method needs a comment about what the input/output are.
Maybe `NdcFromUv` ?

Reply:
It's not NDC either; NDC uses -1…1 coordinates, here it's 0…1.

Reply:
Yeah this isn't NDC. It's a left hand, normalized screen space. I'm fine with using "screenUV" as the terminology, but I'm not sure how to abbreviate it. `suv`?